example/jss/covid/toy/Italy.bl

```diff
@@ -0,0 +1,101 @@
+package blang.project.toy 
+
+model Italy{
+  param Integer alpha ?: 10
+  param Integer smin ?: -1
+  param Integer smax ?: 1
+  param Matrix Sigmad ?: identity(11)
+  random RealVar lambda
+  random List<RealVar> intercept ?: latentRealList(11)
+  param Matrix Sigma ?: {
+  	return identity(11) => [for (int i : 0..<9){
```

Comment:
Code style: Looks like the indentation is inconsistent, sometimes tab, sometimes spaces. Better to avoid it in a given file, makes it harder to read code.